### ISSUE TITLE
Loading dialog sometimes throws exception.

### ISSUE THREAD

**Issue by RadekManak** [pmikova/java-runtime-decompiler#33]
Loading dialog frame is sometimes null when you try to call setvisible(false) on it.

```
Error parsing gtk-icon-sizes string: ''
Exception in thread "Thread-1" java.lang.NullPointerException
	at org.jrd.frontend.MainFrame.VmDecompilerInformationController.hideLoadingDialog(VmDecompilerInformationController.java:146)
	at org.jrd.frontend.MainFrame.VmDecompilerInformationController.loadClassNames(VmDecompilerInformationController.java:175)
	at org.jrd.frontend.MainFrame.VmDecompilerInformationController.lambda$changeVm$8(VmDecompilerInformationController.java:108)
	at java.lang.Thread.run(Thread.java:748)
```

**Comment by AurumTheEnd** [comment]
Unable to replicate this in today's version of JRD nor in January 2019's version when checking-out.

My guess for the cause would be agent finishing loading classes and thus calling `hideLoadingDialog()` with `setvisible(false)` in it **before** `SwingUtilities.invokeLater(() -> { loadingDialog = new LoadingDialog(); ... }` gets to execute on the AWT event dispatching thread.

Will nonetheless fix in the near future.

**Comment by AurumTheEnd** [comment]
Fixed in ac60adf2840e6ee04a6699be54f382a64e51e56c.
